tmail-backend/integration-tests/jmap/jmap-integration-tests-common/src/main/scala/com/linagora/tmail/james/common/CalendarEventCounterAcceptMethodContract.scala

```diff
@@ -0,0 +1,744 @@
+    server.getProbe(classOf[DelegationProbe]).addAuthorizedUser(bobCredential.username, aliceCredential.username)
+
+    // When: Alice accepts the counter event on behalf of Bob
+    val response: String =
+      `given`(baseRequestSpecBuilder(server)
+        .setAuth(authScheme(aliceCredential))
+        .addHeader(ACCEPT.toString, ACCEPT_RFC8621_VERSION_HEADER)
+        .build())
+        .basePath("/jmap")
+        .body(
+          s"""{
+             |  "using": [
+             |    "urn:ietf:params:jmap:core",
+             |    "com:linagora:params:calendar:event"],
+             |  "methodCalls": [[
+             |    "CalendarEventCounter/accept",
+             |    {
+             |      "accountId": "$bobAccountId",
+             |      "blobIds": [ "$counterEventBlobId" ]
+             |    },
+             |    "c1"]]
+             |}""".stripMargin)
+      .when
+        .post
+      .`then`
+        .statusCode(SC_OK)
+        .contentType(JSON)
+        .extract
+        .body
+        .asString
+
+    // Then: The response should indicate that the event was successfully accepted
+    assertThatJson(response)
+      .withOptions(IGNORING_ARRAY_ORDER)
+      .inPath("methodResponses[0]")
+      .isEqualTo(
+        s"""[
+           |  "CalendarEventCounter/accept",
+           |  {
+           |    "accountId": "$bobAccountId",
+           |    "accepted": [ "$counterEventBlobId" ]
+           |  },
+           |  "c1"
+           |]""".stripMargin)
+  }
+
+  @Test
+  def shouldReturnNotUpdatedWhenInvalidIcsPayload(): Unit ={
+    val uploadBlobId: String = `given`
+      .basePath("")
+      .header(ACCEPT.toString, ACCEPT_RFC8621_VERSION_HEADER)
+      .contentType("text/plain")
+      .body("invalid")
+    .when
+      .post(s"/upload/$bobAccountId")
+    .`then`
+      .statusCode(SC_CREATED)
+      .extract
+      .path("blobId")
+
+    val response: String =
+      `given`
+        .body(
+          s"""{
+             |  "using": [
+             |    "urn:ietf:params:jmap:core",
+             |    "com:linagora:params:calendar:event"],
+             |  "methodCalls": [[
+             |    "CalendarEventCounter/accept",
+             |    {
+             |      "accountId": "$bobAccountId",
+             |      "blobIds": [ "$uploadBlobId" ]
+             |    },
+             |    "c1"]]
+             |}""".stripMargin)
+      .when
+        .post
+      .`then`
+        .statusCode(SC_OK)
+        .contentType(JSON)
+        .extract
+        .body
+        .asString
+
+    assertThatJson(response)
+      .withOptions(IGNORING_ARRAY_ORDER)
+      .inPath("methodResponses[0]")
+      .isEqualTo(
+        s"""[
+           |    "CalendarEventCounter/accept",
+           |    {
+           |        "accountId": "$bobAccountId",
+           |        "accepted": [],
+           |        "notAccepted": {
+           |            "$uploadBlobId": {
+           |                "type": "invalidPatch",
```

Comment:
Is it really an invalidPath?

Because the patch have no issues. And technically we are not patching. It's not /update.

invalidArguments maybe?